test/TestCliRolePaths.py

```diff
@@ -24,7 +24,7 @@ def run_ansible_lint(self, cwd, bin, role_path, env=None):
 
     def test_run_single_role_path_no_trailing_slash(self):
         cwd = self.local_test_dir
-        bin = '../bin/ansible-lint'
+        bin = 'ansible-lint'
```

Comment:
I think it's better to implement something like `f'{sys.executable} -m ansiblelint'` invocation style